.github/workflows/merge-gate.yml

```diff
@@ -0,0 +1,35 @@
+name: Merge-Gate
+
+on:
+  workflow_call:
+    # Manually added AWS Secrets here
+    secrets:
+      AWS_CLOUDFORMATION_VALIDATOR_ACCESS_KEY_ID:
+        required: true
+      AWS_CLOUDFORMATION_VALIDATOR_ACCESS_KEY_SECRET:
+        required: true
+      AWS_CLOUDFORMATION_VALIDATOR_DEFAULT_REGION:
+        required: true
```

Comment:
Could this maybe be obsolete?
The current workflow is triggered on `workflow_call` and calls `slow-checks.yml` (again as `workflow_call`).
So maybe `merge-gate.yml` does not need to specify _which_ secrets are expected. 
But in `slow-checks.yml` this could be helpful.

Reply:
True, I removed it.